mobile/src/main/java/org/openhab/habdroid/ui/MainActivity.kt

```diff
@@ -897,7 +897,8 @@ class MainActivity : AbstractBaseActivity(), ConnectionFactory.UpdateListener {
             inServerSelectionMode = false
         }
 
-        drawerModeSelectorContainer.isVisible = connection !is DemoConnection
+        drawerModeSelectorContainer.isVisible =
+            connection !is DemoConnection && prefs.getConfiguredServerIds().isNotEmpty()
```

Comment:
Wenn probably also don't need it if there's only one server configured.

Reply:
I should be looked at the `if` right above this. I'll simplify the code.